### DISCUSSION TITLE
Mamba MK4 F22 Mini - firmware support for 2022B

### DISCUSSION THREAD

**Discussion by markdelonowyoyo** [iNavFlight/inav#8345] [Q&A] [answered]
Does any one know how I can get iNav firmware to support/configure the newer Mamba MK4 F22 Mini - 2022B which uses the ICM42688P Gyro ? Older 2022A models which uses BMI270.

Everything on this mini stack works including internal SPL06 barometer with external GPS/compass but the gyro is the limiting factor.  BetaFlight does support this setup however.
Cheers

**Comment by JulioCesarMatias** [comment, marked as answer]
at the moment, unfortunately we don't have the drivers for this IMU. But in the future for sure some dev should implement it. Sorry

**Reply by markdelonowyoyo** [reply]
Thanks for a speedie response...With all of the recent supply chain challenges I think we will see more IMU changes - unfortunately I didn't get a choice when I purchased they just get shipped with different variants now - will park this flight controller until support is provided (I hope for iNav) as my drone skills are likely to lose it with beta flight ;-)

**Comment by MaiaNunes** [comment]
Hello, I have the same issue on my new HAKRCF722DM mini (20x20). HAKRC changed the gyro to ICM42688P and used INAV target (MATEKF722SE) does not recognize it.
There is an BETAFLIGHT target that works fine HAKRCF722D.
Is there a way to pickup the settings from BF and custom INAV target? 
I've tried but my limited knoledge to do this did not help me so far.

**Comment by DzikuVx** [comment]
Mamba 2022B targets should be supported soon

**Comment by MaiaNunes** [comment]
Thank you Paweł, looking forward to that day :)
